src/Dapr.Jobs/Dapr.Jobs.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+      <TargetFrameworks>net6;net7;net8</TargetFrameworks>
```

Comment:
Nit: only the Workflow library explicitly targets `net7` (for some historical reason). I would remove this so it gets picked up by the common build properties.  (The same goes for the version-related properties.)